### PR TITLE
feat: Win vhd release pipeline

### PR DESCRIPTION
**Pull request by marosset**
**Reason for Change**:

Part 3 of changes needed to automate VHD publishing to azure marketplace as described in #2528

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by marosset** [comment]
This script will be run as part of an azure devops release pipeline and will consume artfacts from a VHD build pipeline and a create marketplace sku pipeline

File: vhd/publishing/add-image-version-to-sku.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+
+cd '/build' || exit
```
Comment (CecileRobertMichon) [inline vhd/publishing/add-image-version-to-sku.sh]: these two lines aren't strictly needed, you could remove them if you wanted

File: vhd/publishing/add-image-version-to-sku.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+EOF
+
+echo "Version info:"
+cat version.json
+
+publisher=$(< $SKU_INFO jq -r ".publisher")
+offer=$(< $SKU_INFO jq -r ".offer")
+sku=$(< $SKU_INFO jq -r ".sku_id")
+
+# TODO: Update pub veresions put to take in version.json as a file
```
Comment (CecileRobertMichon) [inline vhd/publishing/add-image-version-to-sku.sh]:
```suggestion
# TODO: Update pub versions put to take in version.json as a file
```

File: vhd/publishing/add-image-version-to-sku.sh (hunk 1 above)
Comment (CecileRobertMichon) [reply vhd/publishing/add-image-version-to-sku.sh]: oh nvm I re-added them in mine

**Pull request merged**